Tests/ApolloCodegenTests/CodeGeneration/Templates/SwiftPackageManagerModuleTemplateTests.swift

```diff
@@ -14,20 +20,20 @@ class SwiftPackageManagerModuleTemplateTests: XCTestCase {
     """
 
     // when
-    let actual = subject.render()
+    let actual = renderSubject()
 
     // then
     expect(actual).to(equalLineByLine(expected, ignoringExtraLines: true))
   }
 
-  func test_boilerplate__generatesRequiredImports() {
+  func test__boilerplate__generatesRequiredImports() {
```

Comment:
These tests don't need to be validating import statement behaviour - will change.